prow/cmd/gerrit/README.md

```diff
@@ -0,0 +1,72 @@
+# Gerrit
+
+[Gerrit] is a free, web-based team code collaboration tool.
+
+## Gerrit component
+
+The gerrit component is a Prow adapter for handling CI on gerrit workflows. It can poll gerrit
+changes from multiple gerrit instances, and trigger presubmits on Prow upon new patchsets on
+gerrit changes.
+
+## Underlying infra
+
+### Client
+
+We have a [gerrit-client package](/prow/gerrit/client) that provides a thin wrapper around  
+[andygrunwald/go-gerrit](https://github.com/andygrunwald/go-gerrit), which is a go client library
+for accessing the [Gerrit Code Review REST API](https://gerrit-review.googlesource.com/Documentation/rest-api.html)
+
+You can create a client instance by pass in a map of instance-name:project-ids, and pass in an oauth token path to
```

Comment:
The user interface for prow/cmd/gerrit is flags.

Maybe move these docs to prow/gerrit/client?

Reply:
good point to write two docs, will split it up

Reply:
Sweet